site/docsource/Installation.adoc

```diff
@@ -16,7 +16,7 @@ is installed, run the following command:
 
 [source,sh]
 ------------------------------
-npm install --save bs-platform
+npm install --save-dev bs-platform
```

Comment:
devDependencies are dependencies used for building the package. Those dependencies are not downloaded when your package depends on a library. The dev dependencies are usually used for transpilers, test framework and documentation builders and I think it's the case for bucklescript.

For reference, NPM doc on the subject https://docs.npmjs.com/files/package.json#devdependencies